### ISSUE TITLE
Val loss and libero eval code

### ISSUE THREAD

**Issue by ReturnZhao** [Max-Fu/otter#5]
Hello, thank you for this work. I reproduced your training process and found that the val loss did decrease at the beginning, but after just a few vals it remained almost the same, while the training loss was still decreasing. Is this correct? Regarding whether libero's eval code can be open source, the code I tested myself did not succeed once.

**Comment by Max-Fu** [comment]
Hi, 

Thanks for your interest in the work! The validation loss may remain the same while the training loss reduces. I haven't tested this torch reimplementation on Libero yet (the results in the paper are done with the jax version). I have only tested it on some of the real robot tasks. Eval code will be released in the next few weeks (mid-April). 

Best,
Max
